addons/l10n_be/data/template/account.account-be.csv

```diff
@@ -210,31 +210,31 @@
 "a61032","Operational leasing (utility cars)","61032","expense","","False","Leasing opérationnel (voitures utilitaires)","Operationele leasing (bedrijfsauto's)","Operatives Leasing (Nutzfahrzeuge)","Leasing fees of utility cars used in the context of your activity","attn_VAT-IN-V82-21-S"
 "a611","Maintenance services","611","expense","","False","Prestations d'entretien","Onderhoudsdiensten","Wartungsdienste","Services related to the maintenance of your worplace, like repair or cleaning fees","attn_VAT-IN-V82-21-S"
-"a611201","Maintenance (cars)","611201","expense","","False","Entretien (voitures)","Onderhoud (auto's)","Wartung (Autos)","Expenses made for the repair and maintenance of a vehicle",""
+"a611201","Maintenance (cars)","611201","expense","","False","Entretien (voitures)","Onderhoud (auto's)","Wartung (Autos)","Expenses made for the repair and maintenance of a vehicle","attn_VAT-IN-V82-D35"
 "a611202","Maintenance (motorbikes)","611202","expense","","False","Entretien (motos)","Onderhoud (motoren)","Wartung (Motorräder)","Expenses made for the repair and maintenance of a vehicle","attn_VAT-IN-V82-21-S"
 "a611203","Maintenance (bikes)","611203","expense","","False","Entretien (vélos)","Onderhoud (fietsen)","Wartung (Fahrräder)","Expenses made for the repair and maintenance of a bike","attn_VAT-IN-V82-21-S"
 "a611204","Maintenance (e-bikes)","611204","expense","","False","Entretien (vélos électriques)","Onderhoud (e-bikes)","Wartung (E-Bikes)","Expenses made for the repair and maintenance of a e-bike","attn_VAT-IN-V82-21-S"
 "a6113","Maintenance (utility cars)","6113","expense","","False","Entretien (voitures utilitaires)","Onderhoud (bedrijfsauto's)","Wartung (Nutzfahrzeuge)","Expenses made for the repair and maintenance of a vehicle","attn_VAT-IN-V82-21-S"
 "a612","Office supplies","612","expense","","False","Fournitures de bureau","Kantoorbenodigdheden","Bürobedarf","Various supplies bought for your workplace or office, like pencils, paper etc","attn_VAT-IN-V82-21-G"
 "a6122","Office services","6122","expense","","False","Services de bureau","Kantoordiensten","Bürodienstleistungen","All expenses made for your office or workplace that are not material goods","attn_VAT-IN-V82-21-S"
 "a6125","Water, electricity, gas","6125","expense","","False","Eau, électricité, gaz","Water, elektriciteit, gas","Wasser, Strom, Gas","Bills from your water and energy suppliers","attn_VAT-IN-V82-21-S"
-"a612901","Fuel (cars)","612901","expense","","False","Carburant (voitures)","Brandstof (auto's)","Kraftstoff (Autos)","All fuel expenses for a car",""
+"a612901","Fuel (cars)","612901","expense","","False","Carburant (voitures)","Brandstof (auto's)","Kraftstoff (Autos)","All fuel expenses for a car","attn_VAT-IN-V82-D35"
```

Comment:
@chklop This VAT is fine if they have a fuel card with an invoice, but not for regular receipts from the gas station. Do we want to keep it like this?